site/app/views/AutoGradingView.php

```diff
@@ -278,7 +278,7 @@ private function autoGetImageSrc(string $path): string {
      * @param array $uploaded_files
      * @return string
      */
-    public function showTAResults(TaGradedGradeable $ta_graded_gradeable, bool $regrade_available, array $uploaded_files) {
+    public function showTAResults(TaGradedGradeable $ta_graded_gradeable, bool $regrade_available, array $uploaded_files, $teamID) {
```

Comment:
You should probably specify the type on your variable unless it's a mixed type.

Reply:
From what I have seen, the $id could return an int or a string depending on whether or not the id is from a team or individual.